kas-fleet-manager/deploy-kas-fleet-manager.sh

```diff
@@ -197,17 +197,17 @@ deploy_kasfleetmanager() {
 - "id": "standard"
   "display_name": "Standard"
   "sizes":
-  - "id": "x1"
+  - "id": "x2"
```

Comment:
@machi1990 should the sizes array include both x1 and x2, and should we have a separate set of env var (ingress etc) for both profiles?

Reply:
In my opinion, the size array should include both sizes, plus the developer one: 

Actually, for this PR (assuming that it goes first before #164) using the default value in https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/main/templates/service-template.yml#L163 will be enough and there is no need to override it here. 

Once #163 is in, this will allow for better configurations fine tuning and the various specific configurations will be done via what #163 exposes.

Reply:
/cc @MikeEdgar what do you think?

Reply:
My thinking is that if we can allow the user to specify fleet-manager parameters directly rather than patching user options into fleet-manager parameters, that will not only give more control to user configuration, but also reduce code changes when options change, etc. Not to mention the duplicate documentation for env that the installer supports. This is started in #163, but can probably be improved before it's merged.

Reply:
I agree, after https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/pull/163 merged I think we can close this PR.